courses-rest/build.gradle

```diff
@@ -1,3 +1,6 @@
+repositories {
+    mavenLocal()
```

Comment:
ditto

Reply:
I added it be able to develop against nva-commons before it was published on central. I will remove it.